.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    reviewers:
+      - abhijeetnarvekar
+  - package-ecosystem: "nuget"
+    directory: "/"
```

Comment:
I guess this directory should be "/Acrolinx.Net/Acrolinx.Net/" instead of "/", @abhijeetnarvekar ?

Reply:
@Thilo-Acrolinx Yes it needs to be /Acrolinx.Net/Acrolinx.Net/

Reply:
Changed, please review again @abhijeetnarvekar